axelrod/result_set.py

```diff
@@ -401,6 +423,13 @@ def _update_cooperation(self, p1, p2, cooperations):
         self.cooperation[p1][p2] += cooperations[0]
         self.cooperation[p2][p1] += cooperations[1]
 
+    def _update_state_distribution(self, p1, p2, counter):
+        self.state_distribution[p1][p2] += counter
+
+        counter[('C', 'D')], counter[('D', 'C')] = (counter[('D', 'C')],
```

Comment:
I like the swap but not the line break.... I guess there's no getting around that, but maybe it would be more clear if broken after the = sign